docker-compose.ci.yml

```diff
@@ -2,9 +2,6 @@ version: '2'
 
 services:
   app:
-    volumes:
-      - .:/app
-      - ~/cache/.m2:/root/.m2
```

Comment:
The Maven cache would be good to restore at some point. The overhead of downloading the same dependencies over and over is high.

Reply:
I think we should _eventually_ split the image in two - one with the base dependencies, and the other one with that PLUS the app's code.